code/modules/arcade/prize_counter.dm

```diff
@@ -53,7 +47,17 @@
 				print_tickets()
 			default_deconstruction_crowbar(user, O)
 		return
-	return ..()
+
+	. = ..()
```

Comment:
```suggestion
	return ..()
```
just leave this as return here, it's the same functionality but if we're terminating a proc will a call to the parent function this is the standard for it